hopscotch-web/src/main/java/charliek/hopscotch/web/HopscotchApp.java

```diff
@@ -28,17 +33,20 @@ public static void main(String[] args) throws Exception {
 			.registry(Guice.registry(b -> b
 				.bindInstance(ConfigData.class, configData)
 				.module(HopscotchModule.class)
```

Comment:
If the hopscotch module is commented out the static files work as expected.  Or if the ApiHandlerDecorator is commented out things also work as expected.